server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v63/PopulateUuidColumnOfEvents.java

```diff
@@ -37,8 +37,7 @@ public PopulateUuidColumnOfEvents(Database db, UuidFactory uuidFactory) {
     this.uuidFactory = uuidFactory;
   }
 
-  @Override
-  public void execute(Context context) throws SQLException {
+  protected void execute(Context context) throws SQLException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1161)